bitsv/network/services/bchsvexplorer.py

```diff
@@ -13,7 +13,7 @@
 BSV_TO_SAT_MULTIPLIER = BSV
 
 
-class BchSVExplorerDotComAPI:
+class BCHSVExplorerAPI:
```

Comment:
good call

Reply:
More I look at it now the more I think Josh was right, the name does help.